[ADD] legal: enterprise and terms pt_BR

samueljlieber opened the pull request:
Task: [#3627279](https://www.odoo.com/web#id=3627279&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

This PR improves the `enterprise_pt_BR.rst` documentation and adds documentation for `terms_of_sale_pt_BR.rst`.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/7003).

samueljlieber commented:
Updated version history, changed a title, and a paragraph in `enterprise_pt_BR.rst` in https://github.com/odoo/documentation/commit/14a63727ab05805b4e9867eba6715c469ea8f134.
